granary/test/testdata/invite.mf2.json

```diff
@@ -3,7 +3,7 @@
   "properties": {
     "url": ["http://example.com/this/invite"],
     "invitee": [{
-      "type": ["h-card", "p-invitee"],
```

Comment:
`p-*` are property names, not types (mf2 parser couldn't ever give this result). 

I added `parent_prop` to the `_to_html` methods so that things like `class="p-author h-card"` and `class="p-invitee h-card"` get set correctly then.